app.json

```diff
@@ -4,7 +4,7 @@
     "description": "This project is really great.",
     "slug": "campo-libre",
     "privacy": "public",
-    "sdkVersion": "25.0.0",
+    "sdkVersion": "24.0.0",
```

Comment:
I downgraded the SDK version of Expo that we were on since version 25 has a known defect for remote JS debugging in Chrome. (It throws a CORS error, and you literally can't remote debug). The downgrade fixes the bug.  Once you pull down the latest, make sure to `$ rm -rf node_modules` and `$ npm install` to get all the new node modules.